frontend/src/pages/components/q&a.js

```diff
@@ -42,7 +42,9 @@ export default function StateTextFields(props) {
     event.preventDefault();
     setError(true);
     console.log(props.nextPageName)
-    props.pages[props.nextPageName].completed = true;
+    if(props.nextPageName != 'home'){
```

Comment:
You may want to consider making a file that stores the names of the pages in an object, somewhat like how an enum works in other languages so you could do pages.HOME (just incase the name changes for whatever reason and then you would only need to change it in one place) but this isn't a blocking issue, just a suggestion